app/forms/idv/profile_form.rb

```diff
@@ -39,16 +53,28 @@ def pii_attributes
     end
 
     def submit(params)
-      params.each { |key, val| pii_attributes[key] = val }
+      initialize_params(params)
       profile.ssn_signature = ssn_signature
       @success = valid?
       result
     end
 
     private
 
-    attr_writer :first_name, :last_name, :phone, :email, :dob, :ssn, :address1,
-                :address2, :city, :state, :zipcode
+    attr_writer :address1,
```

Comment:
if these are the exact same attrs as we have in `profile_params`, wondering if they should be in a constant somewhere to avoid duplication and make it easier to add an attr....